modules/projects/server/controllers/project.controller.js

```diff
@@ -214,12 +214,19 @@ module.exports = DBModel.extend ({
           var theNode = root.first(function (node) {
             return node.model.id === parseInt(parentId);
           });
+
           // If we found it, add it
           if (theNode) {
           // Check if this already exists.
             var bFound = theNode.first(function (node) {
-            // NB: Exclude myself
-              return (node.model.name === folderName) && (node.model.id !== theNode.model.id);
+              var nodeParentId = '';
+              if(node.parent) {
+                nodeParentId = node.parent.model.id;
+              }
+              // eslint-disable-next-line no-console
```

Comment:
Please remove the `console.log`. It's fine for local debugging but shouldn't be committed to version control...

Reply:
Sorry my bad. Done.